extensions/panache/hibernate-reactive-rest-data-panache/deployment/pom.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-hibernate-reactive-rest-data-panache-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-hibernate-reactive-rest-data-panache-deployment</artifactId>
+    <name>Quarkus - Hibernate ORM Reactive data with Panache - Deployment</name>
+
+    <!-- Defaults, to simplify local testing -->
+    <properties>
+        <postgres.image>postgres:14.1</postgres.image>
+        <postgres.reactive.url>vertx-reactive:postgresql://localhost:5432/hibernate_orm_test</postgres.reactive.url>
+    </properties>
```

Comment:
Is this really needed as the same exist in a profile ?

Reply:
This is how is set in other projects. And I like this way so we can read the properties being in use in the tests right from the beginning.